packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -18,220 +17,233 @@ import BackButtonIcon from 'Assets/ic-poi-back-btn.svg';
 import IDVForm from 'Components/forms/idv-form';
 import PersonalDetailsForm from 'Components/forms/personal-details-form';
 import FormSubHeader from 'Components/form-sub-header';
+import { observer, useStore } from '@deriv/stores';
+import FormikConfirmationCheckbox from 'Components/formik-confirmation-checkbox';
+
+type TIdvDocumentSubmit = {
+    handleBack?: () => void;
+    handleViewComplete?: () => void;
+    is_from_external?: boolean;
+    selected_country?: object;
```

Comment:
Could you please specify the type for selected_country?

Reply:
You can use the type from @deriv/api-types `ResidenceList[0]`